src/librustc_error_codes/error_codes/E0761.md

```diff
@@ -0,0 +1,25 @@
+Multiple candidate files were found for an out-of-line module.
+
+Erroneous code example:
+
+```rust
+// file: ambiguous_module/mod.rs
+
+fn foo() {}
+```
+
+```rust
+// file: ambiguous_module.rs
+
+fn foo() {}
+```
+
+```ignore (compile_fail not working here; see Issue #43707)
```

Comment:
#43707 is closed. Should this be https://github.com/rust-lang/rust/issues/45738, which seemingly replaced #43707?

Reply:
I took the note pointing at #43707 from `E583`, should I update that note as well?

Reply:
Doing a little digging, I don't think either is relevant to #45738 so I've updated